frontend/talentsearch/src/js/components/governmentInfoForm/GovernmentInfoForm.test.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * @jest-environment jsdom
+ */
+import "@testing-library/jest-dom";
+import { render, screen, fireEvent, act } from "@testing-library/react";
+import React from "react";
+import { IntlProvider, MessageFormatElement, useIntl } from "react-intl";
+import GovInfoFormContainer from "./GovernmentInfoForm";
+
+const renderWithReactIntl = (
+  component: React.ReactNode,
+  locale?: "en" | "fr",
+  messages?: Record<string, string> | Record<string, MessageFormatElement[]>,
+) => {
+  return (
+    <IntlProvider locale={locale || "en"} messages={messages}>
+      {component}
+    </IntlProvider>
+  );
+};
+
+let container: HTMLDivElement;
+beforeEach(() => {
+  // setup a DOM element as a render target
+  container = document.createElement("div");
+  document.body.appendChild(container);
+});
+
+jest.setTimeout(30000);
```

Comment:
Experimenting with jest, trying to make things work, like make the component load properly before testing starts
First time using jest, so assume cluelessness in these cases

Reply:
is this resolved? if so, should we mark these as such for subsequent reviewers?

I'd personally love if all comments were marked "resolved" before merge, but unsure who's responsible for closing them (commenter or PR assignee).

@vd1992 might you be able to do a pass and close all the things you understand as resolved? I feel skittish to do that. Esp bc I don't have many reviews under my belt with this team.

my general assumption is that things marked as "resolved" might be re-opened for cursory review, but that i can generally skip over, no harm no foul. And so comments still open are things still awaiting input or a decision. So I wonder if closing could be an opportunity to communicate something between team members, that we're currently leaving ambiguous. ie. can i skip this and think absolutely nothing about it? If someone knew "yes, this is skippable" before my review, is there a way to prevent someone else having to think about it?

(Do you think it's worth adding a pin about this as a "definition of done" convo on upcoming http://go.talent.c4nada.ca/retro-board ?)

Reply:
@patcon I think your assumptions are correct. Its up to either the reviewer (me) or the PR author (@vd1992) to resolve the convo. If the author resolves a convo they think is addressed, its up to the original reviewer to double check. Other reviewers are free to comment on open convos, or even closed ones if they happen to find one they think shouldn't have been completely resolved.